core/tx_list.go

```diff
@@ -366,17 +373,21 @@ func (l *txList) Add(tx *types.Transaction, priceBump uint64) (bool, *types.Tran
 		if cost := tx.Cost(); l.nativecostcap.Cmp(cost) < 0 {
 			l.nativecostcap = cost
 		}
-		if gasPrice := tx.GasPrice(); l.nativegaspricefloor == nil || l.nativegaspricefloor.Cmp(gasPrice) > 0 {
```

Comment:
Sorry, I missed it why was it removed?

Reply:
It is never actually used anywhere, so it can be removed as a cleanup.

Would be nice to have such changes in a separate commit or even PR to make it clearer that the change doesn't have anything to do with CIP-66. (just as a note, no change needed for this PR)